Runtimes/Supplemental/cmake/modules/FindSwiftCore.cmake

```diff
@@ -52,112 +52,102 @@ endif()
 
 include(FindPackageHandleStandardArgs)
 
+if(NOT DEFINED SwiftCore_USE_STATIC_LIBS)
```

Comment:
I think that we should use `BUILD_SHARED_LIBS` to key this off of. I don't know if we want to have the mixed model ever supported and introducing another variable seems unnecessary.

Reply:
This is used by existing `Find*.cmake` modules defined in CMake itself for allowing folks to control how they want to pull things in. e.g: https://github.com/Kitware/CMake/blob/7d5831424b527207b08add72b8f812fb6bee5049/Modules/FindGLEW.cmake#L101-L102

I asked for this change.

Reply:
I do agree that we should be able to default it with `BUILD_SHARED_LIBS` though.

Reply:
I have struggled to come up with a good pattern here -- I was tempted to use a cache variable, but the FindGLEW find modules and others seem to avoid those (because I think caches should be managed by the project, not by modules)

I am defaulting it based on `BUILD_SHARED_LIBS` two lines below -- I will see if there is a better pattern.

Reply:
Talked with Evan offline, for now this approach is fine -- we can revisit this in the future in case it does not suit future needs.